### ISSUE TITLE
Padded Blocks "never" false positive with ASI and a comment at end of block

### ISSUE THREAD

**Issue by Flet** [eslint/eslint#2397]
This is throwing a lint error in v0.20.0

```js
/* globals window */
/* eslint padded-blocks: [2, "never"], semi: [2, "never"] */
if (window.foo) {
  window.bar = 1
  // comment
}
```
```
6:1 - Block must not be padded by blank lines.
```

This looks to be related to this fix: #2273

**Comment by btmills** [comment]
Looks like a duplicate of #2336

**Comment by Flet** [comment]
Ah, thanks!
